MRML/vtkIGTLToMRMLPolyData.cxx

```diff
@@ -380,10 +380,9 @@ int vtkIGTLToMRMLPolyData::MRMLToIGTL(unsigned long event, vtkMRMLNode* mrmlNode
     
     //------------------------------------------------------------
     // Allocate Status Message Class
-    if (this->OutPolyDataMessage.IsNull())
-      {
-      this->OutPolyDataMessage = igtl::PolyDataMessage::New();
-      }
+    // Overwrite old message because we want to clear arrays from previous
+    // calls that will not be set anew.
+    this->OutPolyDataMessage = igtl::PolyDataMessage::New();
```

Comment:
Thanks for reviewing the changes and picking up this change.

The original code tries to avoid repeated New() calls, when we stream messages to the network. It won't make a difference for PolyData messages, which are usually sent for a few times in a session, but can be significant for some other message types such as transforms and images, which are often streamed to the network at 10-100fps.

So, I would stay with the old code to keep the class consistent with other message types, unless these lines cause some issues. Was there any issue caused by the original code?